package.json

```diff
@@ -37,7 +37,8 @@
     "dist": "tsc && electron-builder",
     "mas": "tsc && electron-builder --mac mas",
     "travis": "cd ./scripts && ./travis-build-test.sh",
-    "build-locales": "node tools/locale-helper"
+    "build-locales": "node tools/locale-helper",
+    "build": "tsc --publish never"
```

Comment:
This should just be `tsc`. The `--publish never` _belongs_ to electron-builder cli hence the suggestion the add a `"publish": "never"` under the `"build"` section in package.json.

Reply:
@priyank-p I modified it.